astroquery/mast/cutouts.py

```diff
@@ -637,3 +637,163 @@ def get_cutouts(self, coordinates, *, size=5, survey=None):
+
+        # Set up the download path
+        path = os.path.join(path, '')
+        zipfile_path = "{}hapcut_{}.zip".format(path, time.strftime("%Y%m%d%H%M%S"))
+
+        # Download
+        self._download_file(astrocut_url, zipfile_path)
+        localpath_table = Table(names=["Local Path"], dtype=[str])
+
+        # Checking if we got a zip file or a json no results message
+        if not zipfile.is_zipfile(zipfile_path):
+            with open(zipfile_path, 'r') as FLE:
+                response = json.load(FLE)
+            warnings.warn(response['msg'], NoResultsWarning)
+            return localpath_table
+
+        if not inflate:  # not unzipping
+            localpath_table['Local Path'] = [zipfile_path]
+            return localpath_table
+
+        print("Inflating...")
+        # unzipping the zipfile
+        zip_ref = zipfile.ZipFile(zipfile_path, 'r')
+        cutout_files = zip_ref.namelist()
+        zip_ref.extractall(path, members=cutout_files)
+        zip_ref.close()
+        os.remove(zipfile_path)
+
+        localpath_table['Local Path'] = [path+x for x in cutout_files]
+        return localpath_table
+
+
+    def get_cutouts(self, coordinates, *, size=5):
+        """
+        Get cutout image(s) around the given coordinates with indicated size,
+        and return them as a list of  `~astropy.io.fits.HDUList` objects.
+
+        Parameters
+        ----------
+        coordinates : str or `astropy.coordinates` object
+            The target around which to search. It may be specified as a
+            string or as the appropriate `astropy.coordinates` object.
+        size : int, array-like, `~astropy.units.Quantity`
+            Optional, default 5 pixels.
+            The size of the cutout array. If ``size`` is a scalar number or
+            a scalar `~astropy.units.Quantity`, then a square cutout of ``size``
+            will be created.  If ``size`` has two elements, they should be in
+            ``(ny, nx)`` order.  Scalar numbers in ``size`` are assumed to be in
+            units of pixels. `~astropy.units.Quantity` objects must be in pixel or
+            angular units.
+
+        Returns
+        -------
+        response : A list of `~astropy.io.fits.HDUList` objects.
+        """
+
+        # Get Skycoord object for coordinates/object
+        coordinates = parse_input_location(coordinates)
+
+        param_dict = _parse_cutout_size(size)
+
+        # Need to convert integers from numpy dtypes
+        # so we can convert the dictionary into a JSON object
+        # in service_request_async(...)
+        param_dict["x"] = int(param_dict["x"])
+        param_dict["y"] = int(param_dict["y"])
+
+        # Adding RA and DEC to parameters dictionary
+        param_dict["ra"] = coordinates.ra.deg
+        param_dict["dec"] = coordinates.dec.deg
+
+        response = self._service_api_connection.service_request_async("astrocut", 
```

Comment:
extreme nitpick: fyi: we allow longer than 80char lines, no need to cut this into 3